components/ResearchHubBanner.js

```diff
@@ -70,12 +71,11 @@ class ResearchHubBanner extends Component {
               {this.props.all || this.props.home
                 ? "ResearchHub"
                 : this.props.hub.name}
-              !
             </span>
           </div>
           <div className={css(styles.subtext, styles.text)}>
-            We're a community seeking to improve prioritization, collaboration,
-            reproducibility, and funding of scientific research.{" "}
+            We are bulding an open platform and community whose goal it is to
+            accelerate science.{" "}
```

Comment:
Having strings represented within react components like below preserves formatting. Just a thing to consider
```suggestion
            {"We are bulding an open platform and community whose goal it is to
            accelerate science. "} 
```